src/components/AppFooter.vue

```diff
@@ -1,21 +1,40 @@
 <template>
-  <footer>
-    <p>
-      Powered by <a :title="name" :href="homepage">{{ name }}</a> {{ version }}
-    </p>
-  </footer>
+  <div id="app-footer">
+    <v-footer>
+      <v-row
+       justify="center"
+       align="center"
+       no-gutters
+      >
+        <v-col
+         class="primary text-center"
+         cols="12"
+        >
+          <p>
+            Powered by <a :title="name_" :href="homepage_">{{ name_ }}</a> {{ version_ }}
+          </p>
+        </v-col>
+        <v-col
+         cols="12"
+        >
+          <img :src="`${require('@/assets/img/wmo-foot.png')}`" />
+        </v-col>
+      </v-row>
+    </v-footer>
+  </div>
 </template>
 
 <script>
 import { name, version, homepage } from "../../package";
 
 export default {
   name: "AppFooter",
+  template: "#app-footer",
   data() {
     return {
-      version: version,
-      name: name,
-      homepage: homepage,
+      version_: version,
```

Comment:
Do the vars not work without the trailing `_` ?

Reply:
Was getting some linting warning about exporting them as default but it does work